src/runtime/particle.ts

```diff
@@ -248,6 +249,19 @@ export class Particle {
     throw new Error('A particle needs a description handle to set a decription pattern');
   }
 
+  // Entity functions.
+  idFor(entity: Entity): string {
+    return Entity.id(entity);
```

Comment:
Personally, I found these helper methods a little confusing at first (when I was looking at the particle JS files above). Is there any reason that we can't just use the `Entity.id()` static method everywhere, and remove these particle helper methods?

Reply:
My knee-jerk reaction is also that `Entity.xx()` statics would be better than API on Particle ... so same question.

Reply:
The runtime Entity class isn't available to Particles (and I don't think we want to change that).